docs/types.rst

```diff
@@ -1335,4 +1334,4 @@ Addresses
 As described in :ref:`address_literals`, hex literals of the correct size that pass the checksum
 test are of ``address`` type. No other literals can be implicitly converted to the ``address`` type.
 
-Explicit conversions from ``bytes20`` or any integer type to ``address`` results in ``address payable``.
\ No newline at end of file
+Explicit conversions from ``bytes20`` or any integer type to ``address`` results in ``address payable``.
```

Comment:
Should be plural ``Explicit conversions... result in...`` instead of "result**s**".
Maybe we could even change it to be more precise, e.g.: "Explicitly converting an expression of type ``bytes20`` or of any integer type using ``address(...)`` results in an expression of type ``address payable``."